app/models/manageiq/providers/ibm_terraform/configuration_manager.rb

```diff
@@ -24,6 +24,7 @@ class ManageIQ::Providers::IbmTerraform::ConfigurationManager < ManageIQ::Provid
   belongs_to :provider, :autosave => true, :dependent => :destroy
 
   has_many :orchestration_stacks, :dependent => :destroy, :foreign_key => "ems_id"
+  has_many :computer_systems,     :dependent => :destroy, :foreign_key => "managed_entity_id"
```

Comment:
so you want to have: `has_many :computer_systems, :through => :configured_systems` because the computer_system isn't actually directly related to the `ext_management_system`

Reply:
Thanks adam, that solved the issue with duplicate Computer system